docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,129 @@
+# ADR 008: Decentralized Computer Emergency Response Team (dCERT) Group
+
+## Changelog
+
+- 2019 Jul 31: Initial Draft
+
+## Context
+
+In order to reduce the number of parties involved with handling sensitive
```

Comment:
This paragraph seems to be discussing responsibilities which are off-chain. Does it need to be in this ADR?

My two cents - ADRs should focus on technical implementations of particular ideas, not on the general merits of the ideas, especially when the ideas concern off-chain social structures.